gds-api-adapters.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'lrucache', '~> 0.1.1'
   s.add_dependency 'rest-client', '~> 1.8.0'
   s.add_dependency 'rack-cache'
+  s.add_dependency 'uk_postcode', '2.1.0'
```

Comment:
In `govuk_message_queue_consumer` we are using Airbrake without requiring it. Perhaps we could  raise an error `unless defined?(UKPostcode)` in the postcode sanitizer?